l10n_vn_account_invoice_number/models/account_invoice.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2006-2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    invoice_number = fields.Integer()
+    invoice_form = fields.Char(size=10)
```

Comment:
I think we can remove the size, why do you specify? In case the number changes to 11 characters we will need to change the code...

I think it is wrong with 10, I have an invoice form with 11 characters: 01GTKT2/009